### DISCUSSION TITLE
How to paste password into server

### DISCUSSION THREAD

**Discussion by QuotoDaimo** [isontheline/pro.webssh.net#1184] [Q&A]
Hi. I am experimenting with WebSSH, and so far it looks good :-)
Currently I am setting up access to some small VPSs. These have passwords that are in a password manager.
I can copy these long passwords from my password manager, but I cannot paste that into the server password field in WebSSH.
Is there a way?
Thx.

**Comment by isontheline** [comment]
Hello @QuotoDaimo 👋 

Thanks for your support 🙏

The paste menu doesn’t appear?

**Reply by QuotoDaimo** [reply]
Complicated :-) I now have a work around. Copy directly from the password manager does not work. But if I copy to a text editor, and then copy from text editor I can paste into WebSSH. Strange. Not sure what the difference is, but it doesn't look like your issue. Thx for your response.

**Reply by isontheline** [reply]
> but it doesn't look like your issue

That means you can't paste on other apps also?

**Reply by QuotoDaimo** [reply]
Hi @isontheline 
I CAN copy:  password manager => text app.
I CAN copy: text app => WebSSH.
I CANNOT copy: password manager => WebSSH.
Note 1: I copied by highlighting text, then press "Copy".
Note 2: it DOES work from password manager to WebSSH if I click the "password copy" icon. However, this is not always available in my password manager.
It is complicated. I do not understand all the iOS options for copy & paste there there seem to be some different options.

**Reply by isontheline** [reply]
Which password manager are you using?

**Reply by QuotoDaimo** [reply]
Last Pass

**Reply by isontheline** [reply]
> However, this is not always available in my password manager.

Not all entries have the "copy" submenu?
